data/presets/railway/rail.json

```
@@ -2,10 +2,12 @@
     "icon": "temaki-railway_track",
     "fields": [
         "{railway/light_rail}"
```

syntax error (missing comma at end of the line here)

```suggestion
        "{railway/light_rail}",
```